Week-2/Week2Assignment1/index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
For Assignment purpose only: add description of the work/assignment you are doing here in description tag, <meta name="description" content="" />